### ISSUE TITLE
remove the use of JObject

### ISSUE THREAD

**Issue by javigomez** [joomla-projects/com_localise#104]
I think we can stop using JObject (see https://github.com/joomla-projects/com_localise/search?q=JObject&ref=cmdform) and use a StdClass instead, isn't it?

@gunjanpatel or @wilsonge are any of you interested in taking this task?

Thanks

**Comment by wilsonge** [comment]
I think probably JRegistry is easier for now because of the sheer number of get and set methods :p 

But there was some use of getProperties I saw which meant we can't do a straight swap in some areas

**Comment by javigomez** [comment]
right JRegistry. ^_^

**Comment by gunjanpatel** [comment]
Yes, it seems. We can't able to directly swap if we will use `JRegistry`but it is a good idea to use it, agree.
